test/html_json.test.js

```diff
@@ -39,10 +39,6 @@ describe('HTML Indexing', () => {
 
   before(async () => {
     testRoot = await createTestRoot();
-    nock('https://raw.githubusercontent.com')
-      .get(/\/helix\/github.com--adobe--helix-index-pipelines\/.*\/helix-query.yaml/)
-      .replyWithFile(200, resolve(specsDir, 'blog', 'helix-query.yaml'))
-      .persist();
     nock('https://github.com--adobe--helix-index-pipelines-helix.project-helix.page')
       .get((uri) => uri.startsWith('//test/specs/blog'))
```

Comment:
why double slash? probably something wrong with the URL

Reply:
Yep, the `fetch` property in `helix-query.yaml` was malformed.